webapp/components/_new/features/Admin/Badges/BadgesSection.spec.js

```diff
@@ -5,14 +5,14 @@ const localVue = global.localVue
 
 const badge1 = {
   id: 'badge1',
-  icon: 'icon1',
```

Comment:
We rely on the leading `'/'` here.

Reply:
I don't like this solution - have a look at https://github.com/Ocelot-Social-Community/Ocelot-Social/pull/8558#discussion_r2195816598